annotate.py

```diff
@@ -53,8 +53,9 @@ def draw_circle(event,x,y,flags,param):
         ## click on image to grab pixel coordinates
         ## s to close image and exit program
         ## r to write coordinates to file (you will be prompted for the PMT feature ID)
-def annotate_img(img_path, size1, size2, initials) :
+def annotate_img(img_path, initials, size1=1, size2=1) :
```

Comment:
just to make sure, do you want this reflected in the file output as well or just the way `annotate_dir()` is called?

Reply:
just the way it's called. so now you don't need to specify brush size, which was only really necessary for ML masks.

can just call `annotate_dir(filename, initials)`